src/app/features/navigation-bar/navigation-bar.component.html

```diff
@@ -16,6 +16,13 @@
           [btnIcon]="this.btnCartIcon"
         ></nav-button>
       </li>
+      <li>
+        <nav-button
+          *ngIf="isUserCustomer === false"
```

Comment:
You should check if the user is an owner instead of check if the user is not a customer. Maybe we add a new role in the future, like 'admin'